src/platform/silabs/CHIPDevicePlatformConfig.h

```diff
@@ -91,11 +91,11 @@
 #endif // CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
 
 #ifndef CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL
-#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL chip::System::Clock::Milliseconds32(SL_SLEEP_TIME_MS)
```

Comment:
This would have to be reverted. It breaks the feature of configuring the sleep time.

Reply:
done